File: VB/CustomAppointmentFlyoutExample/MyFlyout.vb
Imports System.ComponentModel
Imports System.Drawing
Imports DevExpress.XtraEditors

Namespace CustomAppointmentFlyoutExample

    Public Partial Class MyFlyout
        Inherits XtraUserControl

        Public Sub New()
            InitializeComponent()
        End Sub

        Public Sub New(ByVal subject As String, ByVal start As Date, ByVal [end] As Date)
            InitializeComponent()
            labelControl1.Text = subject
            labelControl2.Text = start.ToString()
            labelControl3.Text = [end].ToString()
        End Sub
    End Class
End Namespace

File: CS/CustomAppointmentFlyoutExample/MyFlyout.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace CustomAppointmentFlyoutExample {
    public partial class MyFlyout : XtraUserControl {

        public MyFlyout() {
            InitializeComponent();
        }

        public MyFlyout(string subject, DateTime start, DateTime end) {
            InitializeComponent();

            this.labelControl1.Text = subject;
            this.labelControl2.Text = start.ToString();
            this.labelControl3.Text = end.ToString();
        }
    }
}
